awx/main/models/__init__.py

```diff
@@ -16,7 +16,7 @@
 from ansible_base.lib.utils.models import user_summary_fields
 
 # AWX
-from awx.main.models.base import BaseModel, PrimordialModel, accepts_json, CLOUD_INVENTORY_SOURCES, VERBOSITY_CHOICES  # noqa
+from awx.main.models.base import BaseModel, PrimordialModel, accepts_json, compute_cloud_inventory_sources, VERBOSITY_CHOICES  # noqa
```

Comment:
With https://github.com/ansible/awx/pull/15537#discussion_r1786342549, I don't think that there should be a possibility to do `from awx.main.models import compute_cloud_inventory_sources`, because it's not a model.
```suggestion
from awx.main.models.base import BaseModel, PrimordialModel, accepts_json, VERBOSITY_CHOICES  # noqa
```